.github/workflows/build.yaml

```
@@ -6,13 +6,23 @@ on:
     - development
 #    - '[0-9]+.[0-9]+.x'
 
+  workflow_dispatch:
+    inputs:
+      docker_registry:
+        description: 'Docker registry to push images to (default: ghcr.io/, use registry.hub.docker.com/ for docker hub)'
+        required: true
+        default: 'ghcr.io/'
+      docker_repo:
+        description: 'Docker repo to push images to (default: lowered github repository owner name)'
```

```suggestion
        description: 'Docker repo to push images to (default: lowercase github repository owner name)'
```
Is that what you mean?

---

Yes